info/v1/machine.go

```diff
@@ -14,6 +14,10 @@
 
 package v1
 
+import (
+	"github.com/google/cadvisor/utils/cloudinfo"
+)
```

Comment:
Sorry, I didn't understand why the data structures should go into machine.go Why shouldn't consumers of the API have to include utils/cloudinfo? Aren't the methods that retrieve information about the cloud provider in utils/cloudinfo anyway? Thanks!